x-team/xp-ui
# Wire up the tooltip ui component with redux in the application

**Release Type:** *Breaking Changes*

Fixes https://x-team-internal.atlassian.net/browse/XP-2929

## Description

Removed DropDown from ApplicantBadge, refactored calculations to arrow and names.

## Checklist


## Related PRs

XP-2929-wire-up-the-tooltip-ui-component-with-redux-in-the-application | [XP](https://github.com/x-team/xp/pull/1849)

## Steps to Test or Reproduce

1. Filter component ListExclusionFormPopup
2. Check if arrows are in place when clicking on different buttons, check if name changes in all Popups

## Impacted Areas in Application

- `ui/ListExclusionFormPopup.js`

## Screenshots

![image](https://user-images.githubusercontent.com/5478945/55026994-f39f0900-4fe2-11e9-9ea0-5247fe447e7f.png)
